### PR TITLE
added TextField behavior

### PR DESCRIPTION
**Pull request by nilsonsasaki**
fixed text height
added change name on send

known issues: 
sometimes textbar goes to left 
need to create validation

### CODE REVIEW

**Comment by nilsonsasaki** [comment]
Fond an issue, it isn't changing player name in room, in lobby it changes normally,

File: lib/ui/settings.dart [hunk 1]
```diff
@@ -78,17 +85,34 @@ class _SettingsScreenState extends State<SettingsScreen> {
...
+                          hasFocus = focus;
+                          if(hasFocus==true){
+                            myController.clear();
+                          } else {
+                            myController.text=gc.myPlayer.name;
+                          }},
+                        child: TextField(
+                          style: kSettingsTextW400,
+                          textAlign: TextAlign.right,
+                          textDirection: TextDirection.rtl,
```
Comment (yuumasato) [inline lib/ui/settings.dart:100]: Does it make sense to set`TextDirection.rtl`?
When I add a space the cursor jumps to the left.

**Comment by yuumasato** [comment]
> Fond an issue, it isn't changing player name in room, in lobby it changes normally,

I know what is the issue, when the player name is changed the app tries to init a new user, rather than just setting the name. I can check and propose a fix for this.

File: lib/ui/settings.dart (hunk 1 above)
Comment (nilsonsasaki) [reply lib/ui/settings.dart:100]: I think i added it trying to solve the trailing space that don't work but it should be removed

found the issue on Flutter github https://github.com/flutter/flutter/issues/40648

**Pull request closed without merging**